recipes-mender/mender/mender_0.1.bb

```diff
@@ -30,18 +30,19 @@ do_compile() {
   PATH="${B}/bin:$PATH"
   export PATH
 
-  cd "${S}"
-  godep go build -o "${B}/mender"
+  cd ${S}
+  godep go build -o ${B}/mender
 }
 
 do_install() {
-  install -d "${D}/${bindir}"
-  install -m 0755 "${B}/mender" "${D}/${bindir}"
-  install -d ${D}/${systemd_unitdir}/system
-  install -m 0644 ${WORKDIR}/mender.service ${D}/${systemd_unitdir}/system
+  install -d ${D}${bindir}
+  install -t ${D}${bindir} -m 0755 \
+          ${B}/mender ${S}/support/mender-device-identity
+
+  install -d ${D}${systemd_unitdir}/system
+  install -m 0644 ${WORKDIR}/mender.service ${D}${systemd_unitdir}/system
```

Comment:
On Fri, May 20, 2016 at 1:28 PM, Kristian Amlie <notifications@github.com>
wrote:

> In recipes-mender/mender/mender_0.1.bb
> <https://github.com/mendersoftware/meta-mender/pull/43#discussion_r64026859>
> :
>
> >  }
> >
> >  do_install() {
> > -  install -d "${D}/${bindir}"
> > -  install -m 0755 "${B}/mender" "${D}/${bindir}"
> > -  install -d ${D}/${systemd_unitdir}/system
> > -  install -m 0644 ${WORKDIR}/mender.service ${D}/${systemd_unitdir}/system
> > +  install -d ${D}${bindir}
> > +  install -t ${D}${bindir} -m 0755 \
> > +          ${B}/mender ${S}/support/mender-device-identity
> > +
> > +  install -d ${D}${systemd_unitdir}/system
> > +  install -m 0644 ${WORKDIR}/mender.service ${D}${systemd_unitdir}/system
>
> Also don't see the point of removing slashes, they cannot harm, makes it
> more readable, and saves you from having to remember which variables end in
> a slash, and which don't.
>
Consistency again. It's not a strict rule but intermediate slashes are
superfluous. If everyone know that's the case, seeing additional slash
raises questions.

-- 
Maciej Borzecki